resources/scheduled_task.rb

```diff
@@ -60,9 +61,24 @@
     start_time         start_time_value
     start_day          new_resource.start_date unless new_resource.start_date.nil?
   end
+
+  if new_resource.onstart
+    windows_task "#{new_resource.task_name}-onstart" do
+      run_level :highest
```

Comment:
This will need to be gated based on chef release since only the latest versions of chef support this.

Reply:
I don't get it. Which property only supported by latest chef release? It was tested with `13.10` and it works like a charm.

Reply:
@tas50 can you give feedback please